src/stores/snapshot-slice.ts

```diff
@@ -145,6 +141,12 @@ const snapshotSlice = createSlice( {
 				( snapshot ) => snapshot.atomicSiteId !== action.payload.atomicSiteId
 			);
 		},
+		setSnapshots: ( state, action: PayloadAction< Snapshot[] > ) => {
+			if ( ! fastDeepEqual( state.snapshots, action.payload ) ) {
+				state.snapshots = action.payload;
+				state.isLoaded = true;
```

Comment:
```suggestion
				state.isInitialSnapshotsLoaded = true;
```
Not directly related to this PR, but I'm thinking that `isLoaded` may be too generic a name. Might be worth clarifying.